### ISSUE TITLE
Fire action with default value when in one-way mode

### ISSUE THREAD

**Issue by cafreeman** [thefrontside/emberx-select#134]
### Ember Version / Ember CLI Version:
ember-cli: 2.7.0
node: 5.12.0
os: darwin x64

### Expected Behavior

When populating an `x-select` with a list of options, such as
```hbs
{{#x-select one-way=true value=selectedValue action=(action (mut selectedValue)) as |xs|}}
  {{#each things as |thing|}}
    {{xs.option value=thing}}
  {{/each}}
{{/x-select
```

I would expect that `selectedValue` would be set to the first item in the `things` array on initial render since the select box itself will show that value as selected.

### Actual Behavior

Instead, the select box does show the correct value, but the underlying value remains untouched until the user manually triggers a change event.

### Steps To Reproduce:
See example above

**Comment by cowboyd** [comment]
Should this be If `selectedValue` is undefined, null, or is not contained in the selected options?

**Comment by Robdel12** [comment]
Released with 2.2.2 https://github.com/thefrontside/emberx-select/commit/69ece008aaf0b795500dcd9e634d7365c6bb6c7a
